### PR TITLE
Add Bug Report Issue form

### PR DESCRIPTION
**Pull request by imkrishnasarathi**
Partially Fixes issue 21

## Changes proposed

This pr adds a bug report template to the issues

Screenshots
image

### Summary by CodeRabbit

- New Feature: Introduced a comprehensive bug reporting form for users. This new feature enhances the bug reporting process by providing a structured template for users to describe the bug, expected behavior, steps to reproduce, bug frequency, priority, and any additional context. This will aid in faster and more efficient resolution of reported issues.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
+name: Bug Report
```
Comment (xkrishguptaa) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:1]: This could be more like "Report an issue"?

Comment (imkrishnasarathi) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:1]: ok. doing it

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    - Should be addressed eventually
+
+- type: textarea
+  id: additional-context
+  attributes:
+    label: Add any other context about the problem here
+  validations:
+    required: true
+
+
```
Comment (xkrishguptaa) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:67]:
```suggestion
```

Extra new line

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    - Needs to be addressed on a reasonable timescale
+    - Should be addressed when able
+    - Should be addressed eventually
+
+- type: textarea
+  id: additional-context
+  attributes:
+    label: Add any other context about the problem here
+  validations:
+    required: true
```
Comment (xkrishguptaa) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:65]:
```suggestion
    required: false
```

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  attributes:
+    label: How frequent do you see this bug?
+    options:
+      - Everytime (Very Frequently)
+      - Less Frequently
+      - Rarely
+  validations:
+    required: true
+
+- type: dropdown
```
Comment (xkrishguptaa) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:49]: Remove this section, maintainers decide priority

**Review by xkrishguptaa — APPROVED** [review]

**Comment by imkrishnasarathi** [comment]
Hey, thank you for merging my PR! If you want to show support, you can donate just a single coffee at [Buy Me A Coffee](https://www.buymeacoffee.com/imkrishnasarathi). Else you can follow me on [Twitter](https://www.twitter.com/CodesKae) and [Instagram](https://www.instagram.com/imkrishnasarathi/). Happy Hacktoberfest!

**Pull request merged**